x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/scenarios/elasticsearch/index.spec.ts

```diff
@@ -11,7 +11,13 @@ import expect from '@kbn/expect';
 import { MessageRole } from '@kbn/observability-ai-assistant-plugin/common';
 import { chatClient, esClient } from '../../services';
 
-describe('elasticsearch functions', () => {
+describe('Elasticsearch functions', () => {
+  // using 'all' for elasticsearch scenarios enables the LLM correctly pick
+  // elasticsearch functions when querying for data
+  before(() => {
+    chatClient.setScopes(['all']);
```

Comment:
Can we make this a property of `complete` instead? I'm concerned about side-effects

Reply:
Sure @dgieselaar 
I'll make this update.

Reply:
This is updated